extensions/openapi-generator/deployment/src/main/resources/templates/api.qute

```diff
@@ -0,0 +1,56 @@
+package {package};
+
+{#for imp in imports}
+import {imp.import};
+{/for}
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.List;
+import java.util.Map;
+import javax.ws.rs.*;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.MediaType;
+import org.jboss.resteasy.annotations.jaxrs.*;
```

Comment:
What are those used for?

If they are used, we will need another version of that template for RESTEasy Reactive.

Reply:
This is just temporary. First thing is bringing all their templates to Qute, then I'll do the proper changes.